### PR TITLE
Add Error Docs "Daemon need dotnet 7.0 and Install not Create the Fil…

### PR DESCRIPTION
**Pull request by jesperrichert**
Add Error Docs "Daemon need dotnet 7.0 and Install not Create the File in /etc/moonlight/config.json

### CODE REVIEW

File: gettingStarted/index.md [hunk 1]
```diff
@@ -35,6 +35,27 @@ Go to Servers => Nodes and add a new node. After adding a node, go to the setup
...
+E: Unable to locate package dotnet-sdk-7.0
+E: Couldn't find any package by glob 'dotnet-sdk-7.0'
+```
+Then you need to install the Dotnet SDK 7.0 
+> ### Link to microsoft: 
+> - https://learn.microsoft.com/de-de/dotnet/core/install/linux-ubuntu-install?pivots=os-linux-ubuntu-2204&tabs=dotnet7
+> - https://learn.microsoft.com/de-de/dotnet/core/install/linux-ubuntu-install?pivots=os-linux-ubuntu-2204&tabs=dotnet7#ubuntu-2204
+
+#### Command - Copy Paste:<br>
+*(Work on Ubuntu 22.4)*
```
Comment (gOOvER) [inline gettingStarted/index.md:53]: This step is not needed on Ubuntu 22.04. Works fine for me withou adding the repos

Comment (tnsjesper) [reply gettingStarted/index.md:53]: > Yes but on some server not... (My Hetzner Ubuntu 22.4 server i must install dotnet 7.0) And some other user have this Error...

Comment (gOOvER) [reply gettingStarted/index.md:53]: > > Yes but on some server not... (My Hetzner Ubuntu 22.4 server i must install dotnet 7.0) And some other user have this Error...

i also use a Hetzner Server and it works fine. With every install now. MB do not blind use the default Ubuntu repos and the the correct one ;)

Comment (tnsjesper) [reply gettingStarted/index.md:53]: Yes but if you Install Ubuntu 22.4 from the console i have this Problem and if you take a look at the discord other users have the same! 
Masu can merge it but doesn't have to. I think it makes sense but that's her decision.

**Comment by Skullians** [comment]
+1 on this

File: gettingStarted/index.md (hunk 1 above)
Comment (gOOvER) [reply gettingStarted/index.md:53]: > Yes but if you Install Ubuntu 22.4 from the console i have this Problem and if you take a look at the discord other users have the same! Masu can merge it but doesn't have to. I think it makes sense but that's her decision.

Thats not a ML problem; edit the source.list after install. The default source.list is crap

**Comment by tnsjesper** [comment]
> +1 on this

On What?

**Comment by Skullians** [comment]
> > +1 on this
> 
> On What?

+1 as in +1 to the PR

**Comment by gOOvER** [comment]
> +1 on this

No, bacause its an Admin problem, not a ML problem

**Comment by Skullians** [comment]
It's not the Admin's fault that it isn't included in the default source list.
Better to mention it than not mention it at all and discourage users from installing it

**Comment by tnsjesper** [comment]
I don't want to discuss such children's stuff! It's their decision and I live with it. I also know that it will be fixed in the v2 release but right now it's just a good tip!

**Comment by gOOvER** [comment]
> It's not the Admin's fault that it isn't included in the default source list. Better to mention it than not mention it at all and discourage users from installing it

**Comment by Skullians** [comment]
I understand that.
But why waste their time searching for documentation concerning it?

**Comment by gOOvER** [comment]
> I don't want to discuss such children's stuff! It's their decision and I live with it. I also know that it will be fixed in the v2 release but right now it's just a good tip!

Thats not childrean stuff. That is the normal job of an admin.

**Comment by Skullians** [comment]
again to my point, still uses up time that would be solved with this PR

**Comment by Masu-Baumgartner** [comment]
Please fix the grammer mistakes. Mine are enough for the documentation  ;). And if you want to add such a section, add it as "Troubleshooting" to the end section of the getting started guide. And provide the direct commands not just links to the microsoft docs to really help the users. The commands i mention are pinned in the #help channel in discord

**Comment by tnsjesper** [comment]
I will change it to "Troubleshooting" and correct the text....

**Comment by tnsjesper** [comment]
I will change it to "Troubleshooting" and correct the text....

**Comment by tnsjesper** [comment]
I have add the "Troubleshooting" and fix my grammer mistake.

**Comment by tnsjesper** [comment]
The Troubleshooting is done.

**Comment by Masu-Baumgartner** [comment]
It is not. The install command is executed by the install script. The issue is that a package repo might miss. The troubleshooting you added wont change anything

**Comment by tnsjesper** [comment]
Ah ok, you fixed it.. Then my troubleshooting is of course unnecessary!

**Comment by Masu-Baumgartner** [comment]
I havent fixed anything about this as on all my test systems there was dotnet sdk 7 in the package sources. the installer has always executed the apt install for the dotnet 7 sdk package

**Comment by tnsjesper** [comment]
Exactly, that's what I mean

**Pull request closed without merging**